README.md

```diff
@@ -1,432 +1,8 @@
-```json
-{
- "unsigned-message-bytes": "<Hex encoded byte array containing the unsigned warp message>",
- "source-address": "<Hex encoding of address that sent the warp message>"
-}
-```
-- If successful, the endpoint will return the following JSON:
-```json
-{
- "transaction-hash": "<Transaction hash that includes the delivered Warp message>",
-}
-```
-
-#### `/health`
-- Takes no arguments. Returns a `200` status code if all Application Relayers are healthy. Returns a `503` status if any of the Application Relayers have experienced an unrecoverable error. Here is an example return body:
-```json
-{
-  "status": "down",
-  "details": {
-    "relayers-all": {
-      "status": "down",
-      "timestamp": "2024-06-01T05:06:07.685522Z",
-      "error": "<List of cb-58 encoded IDs for unhealthy relayers>"
-    }
-  }
-}
-```
-
-## Testing
-
-### Unit Tests
-
-Unit tests can be ran locally by running the command in the root of the project:
-
-```bash
-./scripts/test.sh
-```
-
-If your temporary directory is not writable, the unit tests may fail with messages like `fork/exec /tmp/go-build2296620589/b247/config.test: permission denied`. To fix this, set the `TMPDIR` environment variable to something writable, for example `export TMPDIR=~/tmp`.
-
-### E2E Tests
-
-E2E tests are ran as part of CI, but can also be ran locally with the `--local` flag. To run the E2E tests locally, you'll need to install Gingko following the instructions [here](https://onsi.github.io/ginkgo/#installing-ginkgo)
-
-Next, provide the path to the `subnet-evm` repository and the path to a writeable data directory (this example uses `~/subnet-evm` and `~/tmp/e2e-test`) to use for the tests:
-
-```bash
-./scripts/e2e_test.sh --local --subnet-evm ~/subnet-evm --data-dir ~/tmp/e2e-test
-```
-
-To run a specific E2E test, specify the environment variable `GINKGO_FOCUS`, which will then look for [test descriptions](./tests/e2e_test.go#L68) that match the provided input. For example, to run the `Basic Relay` test:
-
-```bash
-GINKGO_FOCUS="Basic" ./scripts/e2e_test.sh --local --subnet-evm ~/subnet-evm --data-dir ~/tmp/e2e-test
-```
-
-The E2E tests use the `TeleporterMessenger` contract deployment transaction specified in the following files:
-
-- `tests/utils/UniversalTeleporterDeployerAddress.txt`
-- `tests/utils/UniversalTeleporterDeployerTransaction.txt`
-- `tests/utils/UniversalTeleporterMessagerContractAddress.txt`
-  To update the version of Teleporter used by the E2E tests, update these values with the latest contract deployment information. For more information on how to deploy the Teleporter contract, see the [Teleporter documentation](https://github.com/ava-labs/teleporter/tree/main/utils/contract-deployment).
-
-### Generate Mocks
-
-[Gomock](https://pkg.go.dev/go.uber.org/mock/gomock) is used to generate mocks for testing. To generate mocks, run the following command at the root of the project:
-
-```bash
-go generate ./...
-```
+1. [AWM Relayer](relayer/README.md)
```

Comment:
Should this be ICM relayer now?

May also be helpful to include a 1 sentence description of each service at the top level here.

Reply:
I'll include descriptions but do we want to hold off on changing the name here until we change the name of the build binary / rest of the documentation / and the whole repo?

Reply:
Ah yeah good call